docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/2.balance-syntax.md

```diff
@@ -2,12 +2,16 @@
 
 The `BALANCE` statements support the load balancing operations of the Nebula Graph Storage services. For more information about storage load balancing and examples for using the `BALANCE` statements, see [Storage load balance](../../8.service-tuning/load-balance.md).
 
+!!! compatibility "Legacy version compatibility"
+
+    The `BALANCE` commands is changed because the zone feature is added in Nebula Graph version 3.0.0, `BALANCE` commands before version 3.0.0 is not compatible.
```

Comment:
```suggestion
    The `BALANCE` commands is changed because the zone feature is added to Nebula Graph version 3.0.0, `BALANCE` commands before version 3.0.0 is not compatible.
```